src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/StreamHelper.cs

```diff
@@ -38,55 +38,14 @@ internal class WebResponseContentMemoryStream : MemoryStream
         /// <param name="initialCapacity"></param>
         /// <param name="cmdlet">Owner cmdlet if any.</param>
         /// <param name="contentLength">Expected download size in Bytes.</param>
-        internal WebResponseContentMemoryStream(Stream stream, int initialCapacity, Cmdlet cmdlet, long? contentLength)
-            : base(initialCapacity)
+        internal WebResponseContentMemoryStream(Stream stream, int initialCapacity, Cmdlet cmdlet, long? contentLength) : base(initialCapacity)
         {
             this._contentLength = contentLength;
             _originalStreamToProxy = stream;
             _ownerCmdlet = cmdlet;
         }
         #endregion
 
-        /// <summary>
-        /// </summary>
-        public override bool CanRead
-        {
-            get
-            {
-                return true;
-            }
-        }
-
-        /// <summary>
-        /// </summary>
-        public override bool CanSeek
-        {
-            get
-            {
-                return true;
```

Comment:
Please revert. It is not call base method so we can break something. The change is not for cleanup PR.